pom.xml

```
@@ -464,6 +464,36 @@
 							<modelNameSuffix>dto</modelNameSuffix>
 						</configuration>
 					</execution>
+					<execution>
+						<id>user-stats-api</id>
+						<goals>
+							<goal>generate</goal>
+						</goals>
+						<configuration>
+							<inputSpec>https://raw.githubusercontent.com/pagopa/pagopa-infra/ec79d63445cd3bdb4bcc731161c4c1a11de00db2/src/domains/ecommerce-app/api/ecommerce-user-stats-service/v1/_openapi.json.tpl</inputSpec>
```

```suggestion
							<inputSpec>https://raw.githubusercontent.com/pagopa/pagopa-infra/ec79d63445cd3bdb4bcc731161c4c1a11de00db2/src/domains/ecommerce-app/api/ecommerce-user-stats-service/v1/_openapi.json.tpl</inputSpec>
```

TODO to be replaced with infra tag after merge